### PR TITLE
Link to show your and user's posts is missing in Statistics tab in profile.php

### PR DESCRIPTION
**Pull request by senky**
In label "Total posts" after total number of your or user's posts, in common viewprofile page, there is a link titled with "Show your posts" if you are on your profile and "Search user's posts" if you are on other's profile. This is, however, missing in profile.php. It should be added there.

http://phpbbsocialnetwork.com/viewtopic.php?f=62&t=4205

### CODE REVIEW

**Comment by Culprit-CZ** [comment]
Kamahl work, than he must

File: root/socialnet/profile.php [hunk 1]
```diff
@@ -545,6 +545,7 @@ function tab_stats($user_id)
 				'POSTS'				 => ($member['user_posts']) ? $member['user_posts'] : 0,
 				'WARNINGS'			 => isset($member['user_warnings']) ? $member['user_warnings'] : 0,
 				'S_WARNINGS'		 => ($auth->acl_getf_global('m_') || $auth->acl_get('m_warn')) ? true : false,
+				'U_SEARCH_USER'		 => ($auth->acl_get('u_search')) ? append_sid("{$phpbb_root_path}search.$phpEx", "author_id=$user_id&amp;sr=posts") : '',
```
Comment (Kamahl19) [inline root/socialnet/profile.php]: There is U_SEARCH_USER in root/profile.php , we can use that.

**Comment by Kamahl19** [comment]
Btw this is not a bug but improvement, because we removed the link for 0.7.0

**Comment by Culprit-CZ** [comment]
MA

**Pull request merged**